ceph/utils.py

```diff
@@ -187,7 +190,8 @@ def create_ibmc_ceph_nodes(
                 node_dict = ceph_cluster.get(node)
                 node_params = params.copy()
                 node_params["role"] = RolesContainer(node_dict.get("role"))
-
+                node_params["id"] = node_dict.get("id") or node
```

Comment:
IMO, we could do `node_params["id"] = node_dict.get("id", node)`... not that the above wouldn't work.